pinot-core/src/test/java/org/apache/pinot/core/util/SchemaUtilsTest.java

```diff
@@ -313,6 +315,24 @@ public void testDateTimeFieldSpec()
     SchemaUtils.validate(pinotSchema);
   }
 
+  @Test
+  public void testColumnNameValidation()
```

Comment:
Let's add test for trailing and leading whitespace as well

Reply:
Also please add a comment stating that currently the test is only for checking whitespaces. Later on if we add more validation on column names, we can corresponding tests here.